types/dayzed/index.d.ts

```diff
@@ -4,7 +4,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import { Component, ReactNode } from 'react';
+import { Component, ReactNode } from "react";
```

Comment:
Here and on L22 is there any reason for the change in quotes? PR is all good otherwise. :+1:

Reply:
Local prettier config accidentally changed to double quotes. Have fixed that up :)